app/views/auctions/_auction.html.erb

```diff
@@ -1,6 +1,8 @@
 <div class="js-auction-view">
   <div class="auction-body">
     <div class="auction-description">
+      <h2>Skills</h2>
```

Comment:
Do we want this header to be displayed if we have not assigned skills to an auction?